core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -390,6 +391,7 @@ object Boot extends IOApp with LazyLogging {
             logger.error("FATAL - failure starting http server", t)
             throw t
         }
+        _ <- IO.never
```

Comment:
It took me a while to realize this is needed as well!
But the reason is, [this](https://github.com/broadinstitute/rawls/blob/c6923a3f94a09568cf82953860ba511809633e50/core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala#L389) returns right away, so boot function will return right away, which means we exit out of the `.use` block for all `Resource` and `ExecutionContext` would be shutdown while akka-http is still running the web application somehow.

I just checked SAM, and I think SAM was fine because it has [this](https://github.com/broadinstitute/sam/blob/develop/src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala#L77), which keeps the app from existing the '.use' block. I guess we could use this same line in Rawls as well, but at the time of debugging, `IO.never` is the first thing comes to mind to prevent the method from returning.